src/librustc_session/lib.rs

```diff
@@ -3,7 +3,6 @@
 
 // Use the test crate here so we depend on getopts through it. This allow tools to link to both
 // librustc_session and libtest.
```

Comment:
Is this comment still correct?

Reply:
Ah, it might not be, good call. I *think* that tools should continue to work with this PR, but I can try and check locally (and if so update this comment) due to the `pub use getopts` below. If we do have problems in practice then we can workaround them I think.